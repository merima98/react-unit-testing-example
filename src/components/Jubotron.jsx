import React from "react";

const Jubotron = (props) => {
  return <div className="jubotron">{props.children}</div>;
};

export default Jubotron;
