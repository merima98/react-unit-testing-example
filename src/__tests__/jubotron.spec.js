import React from "react";
import Jubotron from "../components/Jubotron";
import { render } from "@testing-library/react";

describe("Should test Jubotron component", function () {
  it("should render component", function () {
    render(<Jubotron />);
  });
});
