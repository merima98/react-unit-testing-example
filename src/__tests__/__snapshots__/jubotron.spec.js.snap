// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test Jubotron component should render component 1`] = `
<div
  className="jubotron"
/>
`;
